.circleci/config.yml

```diff
@@ -60,6 +60,9 @@ jobs:
           root: ~/rpm
           paths:
             - x86_64
+      - store_artifacts:
+          path: ~/rpm
+          destination: rpm
```

Comment:
I think because the artifact already has his name we don't need the `rpm` prefix